### ISSUE TITLE
Are there any way to change mappings to <C-w><char>

### ISSUE THREAD

**Issue by asyncee** [knubie/vim-kitty-navigator#5]
Hello, i have installed this plugin and it is working fine, thank you!

I want to use default vim mappings `Ctrl+W H/J/K/L`. Is this possible?

I have reconfigured kitty/neovim mappings (as it said in readme) from `ctrl+h` to `ctrl+w>h` (for example), but it is not working.

Thanks!

**Comment by knubie** [comment]
@asyncee I've not tested key chords with the plugin, but I'll take your word that they don't work. If I had to guess I would suspect that the `pass_keys.py` file needs to be updated to support them. Unfortunately I don't know enough about kitty scripting to make that change, but I will happily accept a PR if you want to tackle this issue.

**Comment by asyncee** [comment]
Alright, I will try to look into that, thank you.

**Comment by hbarcelos** [comment]
I could make chords work like this in my `kitty.conf`:

```
map ctrl+b>j kitten vim-kitty-navigator/pass_keys.py neighboring_window bottom ctrl+j
map ctrl+b>k kitten vim-kitty-navigator/pass_keys.py neighboring_window top    ctrl+k
map ctrl+b>h kitten vim-kitty-navigator/pass_keys.py neighboring_window left   ctrl+h
map ctrl+b>l kitten vim-kitty-navigator/pass_keys.py neighboring_window right  ctrl+l
```

I'm not using `ctrl+w` because I like to keep the default functionality (deleting a word from the prompt), but I don't see why it wouldn't work.

**Comment by shiny-comic** [comment]
For someone looking for this solution.

setting the `kitty.conf` like below
```
map ctrl+w>h kitten pass_keys.py neiboring_window left   ctrl+h
map ctrl+w>j kitten pass_keys.py neiboring_window bottom ctrl+j
map ctrl+w>k kitten pass_keys.py neiboring_window top    ctrl+k
map ctrl+w>l kitten pass_keys.py neiboring_window right  ctrl+l
```
and leaving the settings of the plugin as default, or explicitly set as 
`let g:kitty_navigator_no_mappings = 0`

I don't know if this is the correct behavior.

**Comment by mikesmithgh** [comment]
I have this working with the fix #40 and the following configuration in Neovim.
`kitty.conf`
```
map ctrl+w>j kitten pass_keys.py neighboring_window bottom 'ctrl+w>j' nvim
map ctrl+w>k kitten pass_keys.py neighboring_window top    'ctrl+w>k' nvim
map ctrl+w>h kitten pass_keys.py neighboring_window left   'ctrl+w>h' nvim
map ctrl+w>l kitten pass_keys.py neighboring_window right  'ctrl+w>l' nvim
```

`vim-kitty-navigator.lua` (via https://github.com/folke/lazy.nvim package manager)
```lua
return {
  'knubie/vim-kitty-navigator',
  enabled = true,
  init = function()
    vim.g.kitty_navigator_no_mappings = 1

    vim.keymap.set('n', '<c-w>h', '<cmd>KittyNavigateLeft<cr>', { noremap = true, silent = true })
    vim.keymap.set('n', '<c-w>j', '<cmd>KittyNavigateDown<cr>', { noremap = true, silent = true })
    vim.keymap.set('n', '<c-w>k', '<cmd>KittyNavigateUp<cr>', { noremap = true, silent = true })
    vim.keymap.set('n', '<c-w>l', '<cmd>KittyNavigateRight<cr>', { noremap = true, silent = true })
  end,
}
```

**Comment by knubie** [comment]
Closed by #40
